x-pack/plugins/stack_connectors/server/connector_types/sentinelone/render.ts

```diff
@@ -24,22 +24,6 @@ export const renderParameterTemplates = (
   const context = variables?.context as Context;
   const alertIds = map(context.alerts, '_id');
 
-  if (params?.subAction === SUB_ACTION.KILL_PROCESS) {
```

Comment:
I was wondering about this render.ts functionality when developing Crowdstrike. I have this impression that we don't need this file at all. Because what it does it applies alerts from context to the actions - but we don't really call these actions because we use the 'response actions'. 
What do you think @paul-tavares  ?

Reply:
I think you are right @tomsonpl . I also don't really know the true porpose of this, but based on the code I made the same assumption.